storage/memory/memory_test.go

```diff
@@ -303,6 +316,65 @@ func TestObjectsLatestAnchor(t *testing.T) {
 	}
 }
 
+func TestObjectsFilter(t *testing.T) {
+	ts, ctx := getTestTriplesFilter(t), context.Background()
+	g, _ := NewStore().NewGraph(ctx, "test")
+	if err := g.AddTriples(ctx, ts); err != nil {
+		t.Fatalf("g.AddTriples(_) failed failed to add test triples with error: %v", err)
```

Comment:
```suggestion
		t.Fatalf("g.AddTriples(_) failed to add test triples with error: %v", err)
```

Reply:
Done for this one and all the others you commented below. Thank you Thiago. 👍